plugins/PrivacyManager/templates/privacySettings.twig

```diff
@@ -7,266 +7,276 @@
-        <input role="yes" type="button" value="{{ 'General_Yes'|translate }}"/>
-        <input role="no" type="button" value="{{ 'General_No'|translate }}"/>
-    </div>
-    <div class="ui-confirm" id="saveSettingsBeforePurge">
-        <h2>{{ 'PrivacyManager_SaveSettingsBeforePurge'|translate }}</h2>
-        <input role="yes" type="button" value="{{ 'General_Ok'|translate }}"/>
-    </div>
-    <div class="ui-confirm" id="confirmPurgeNow">
-        <h2>{{ 'PrivacyManager_PurgeNowConfirm'|translate }}</h2>
-        <input role="yes" type="button" value="{{ 'General_Yes'|translate }}"/>
-        <input role="no" type="button" value="{{ 'General_No'|translate }}"/>
-    </div>
-    <h2 id="deleteLogsAnchor">{{ 'PrivacyManager_DeleteDataSettings'|translate }}</h2>
-    <p>{{ 'PrivacyManager_DeleteDataDescription'|translate }} {{ 'PrivacyManager_DeleteDataDescription2'|translate }}</p>
-    <form method="post" action="{{ {'action':'saveSettings','form':'formDeleteSettings','token_auth':token_auth} | urlRewriteWithParameters }}" id="formDeleteSettings">
-        <table class="adminTable" style='width:800px;'>
-            <tr id='deleteLogSettingEnabled'>
-                <td width="250">{{ 'PrivacyManager_UseDeleteLog'|translate }}<br/>
+            <h2 id="deleteReportsConfirm">{{ 'PrivacyManager_DeleteReportsConfirm'|translate }}</h2>
 
-                </td>
-                <td width='500'>
-                    <input id="deleteEnable-1" type="radio" name="deleteEnable" value="1" {% if deleteData.config.delete_logs_enable == '1' %}
-                        checked {% endif %}/>
-                    <label for="deleteEnable-1">{{ 'General_Yes'|translate }}</label>
+            <h2 id="deleteBothConfirm">{{ 'PrivacyManager_DeleteBothConfirm'|translate }}</h2>
+            <input role="yes" type="button" value="{{ 'General_Yes'|translate }}"/>
+            <input role="no" type="button" value="{{ 'General_No'|translate }}"/>
+        </div>
+        <div class="ui-confirm" id="saveSettingsBeforePurge">
+            <h2>{{ 'PrivacyManager_SaveSettingsBeforePurge'|translate }}</h2>
+            <input role="yes" type="button" value="{{ 'General_Ok'|translate }}"/>
+        </div>
+        <div class="ui-confirm" id="confirmPurgeNow">
+            <h2>{{ 'PrivacyManager_PurgeNowConfirm'|translate }}</h2>
+            <input role="yes" type="button" value="{{ 'General_Yes'|translate }}"/>
+            <input role="no" type="button" value="{{ 'General_No'|translate }}"/>
+        </div>
+        <h2 id="deleteLogsAnchor">{{ 'PrivacyManager_DeleteDataSettings'|translate }}</h2>
+        <p>{{ 'PrivacyManager_DeleteDataDescription'|translate }} {{ 'PrivacyManager_DeleteDataDescription2'|translate }}</p>
+        <form method="post" action="{{ {'action':'saveSettings','form':'formDeleteSettings','token_auth':token_auth} | urlRewriteWithParameters }}" id="formDeleteSettings">
```

Comment:
`token_auth` set here in action as well which causes it to be sent as GET parameter?